### DISCUSSION TITLE
Random chunks get generated with delimiter.

### DISCUSSION THREAD

**Discussion by insubuydev** [webpack/webpack#17260] [Q&A]
**Do you want to request a _feature_ or report a _bug_?**

**What is the current behavior?**
using splichunks creates dynamic chunks from few files and generates with delimiter, though I have specifically said the chunks should only be in vendor or common in the cacheGroups. Example:  new chunks get generated like "file-one-example ~ file-five-example ~ file-random.js

**If the current behavior is a bug, please provide the steps to reproduce.**

1. In an existing project where the bundling operation is performed use the following splitchunks structure.

`optimization: {

    splitChunks: {
        chunks: 'all',
        cacheGroups: {
            vendor: {
                name: 'vendors',
                chunks: 'all',
                test: /[\\/]node_modules[\\/]/,
                reuseExistingChunk: true,
            },
            common: {
                chunks: 'all',
                minChunks: 9,
                name: 'commons',
                reuseExistingChunk: true,
            }
        }
    }
}`

**What is the expected behavior?**
There is a list of entry files in which the webpack performs the operation. While it does all the nodemodules related chunks should be put into vendors and other shared files/functions shared in atleast 9 diff files should be put in commons. No new chunk file is expected.

**other relevant information**
Latest version of webpack.
OS: Latest windows version.

**Comment by alexander-akait** [comment]
I don't see problems here, you can use `filename` if you want to have a name of group

**Reply by insubuydev** [reply]
No, it should not generate a new file, all the chunks should be put into vendors or commons based on the test. Am I missing any flag here or I even tried minSize in the parent, which omitted the new files but it is missing in the vendors.js

**Reply by alexander-akait** [reply]
Please provide reproducible test case and I will show you what you missing

**Reply by alexander-akait** [reply]
Note webpack has defaultGroups, you can disable it using `defaultVendors: false` and `default: false` - https://webpack.js.org/plugins/split-chunks-plugin/#optimizationsplitchunks
